test/tools/libtesteth/Options.cpp

```diff
@@ -205,6 +206,11 @@ Options::Options(int argc, char** argv)
 		}
 		else if (arg == "--createRandomTest")
 			createRandomTest = true;
+		else if (arg == "--seed")
+		{
+			throwIfNoArgumentFollows();
+			randomTestSeed = argv[++i];
```

Comment:
I think the seed should be a number, not a string. Preferably `uint64_t` as [mt19937_64](http://en.cppreference.com/w/cpp/numeric/random/mersenne_twister_engine) is good choice for random engine.